src/rsz/src/Resizer.cc

```diff
@@ -914,6 +917,13 @@ Resizer::replaceCell(Instance *inst,
 {
   const char *replacement_name = replacement->name();
   dbMaster *replacement_master = db_->findMaster(replacement_name);
+
+  // Legalize the position of the instance in case it leaves the die
+  if (opendp_ && parasitics_src_ == ParasiticsSrc::global_routing)
```

Comment:
When would opendp_ be null?

Reply:
The check is not useful because opendp_ is defined when resizer is initialized, this was fixed in the last commits.